dev-docs/bidders/revnew.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: bidder
+title: RevNew
+description: Prebid RevNew Bidder Adapter
+pbjs: true
+pbs: true
+biddercode: revnew
+tcfeu_supported: true
+usp_supported: true
+gpp_supported: true
```

Comment:
```suggestion
```

`gpp_supported` is deprecated. Use `gpp_sids` with the values you support. 

Possible values:

```
gpp_sids: tcfeu, tcfca, usnat, usstate_all, usp
```